modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/PersistentPageMemoryDataRegion.java

```diff
@@ -35,6 +36,19 @@
  * Implementation of {@link DataRegion} for persistent case.
  */
 class PersistentPageMemoryDataRegion implements DataRegion<PersistentPageMemory> {
+    /**
+     * Threshold to calculate limit for pages list on-heap caches.
```

Comment:
It is not clear which ratio it limited by this. Can this be reformulated as 'maximum fraction of all pages that can be taken by page list caches'?

Also, why does it say about on-heap caches, while page memory is off-heap? Does it mean that pages list caches are always stored on-heap, so these 10% are not taken from the page-memory pages, but they are added on top of them, so memory-wise, this means up to 110% of the memory configured for page-memory?

Reply:
Pages list caches is on-heap this is an optimization from 2.0.
why 110?

Reply:
Tried adding a description.